cypress/e2e/ModelizerPage/SwitchView/delete.feature

```diff
@@ -0,0 +1,228 @@
+    And  I expect "[data-cy=\"plugin-definitions-leto-modelizer-plugin-test\"] [data-cy=\"plugin-definitions-title\"]" is "leto-modelizer-plugin-test"
+    
+    When I click on "[data-cy=\"plugin-definitions-leto-modelizer-plugin-test\"]"
+    Then I expect ".plugin-definitions" appear 1 time on screen
+    And  I expect ".component-definition-card" appear 7 times on screen
+
+  @skip
+    # TODO: Remove @skip tag when https://github.com/ditrit/leto-modelizer/issues/128 is done
+  Scenario: Delete a component (Model view) should remove plugin file (Text view)
+    Given I click on "[data-cy=\"component-definition-image\"]"
+    And  I expect "[id^=\"image_\"]" exists
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "default.plugin-test.js"
+    And  I expect active file content to contain "\"type\": \"image\""
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Model"
+    And  I expect "[data-cy=\"modelizer-model-view-draw-root\"]" exists
+
+    When I click on "[id^=\"image_\"]"
+    And  I click on "[id=\"remove-component\"]"
+    Then I expect "[id^=\"image_\"]" not exists
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    But  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" not exists
+
+  Scenario: Delete one of two components (Model view) should remove corresponding object inside plugin file content (Text view)
+    Given I click on "[data-cy=\"component-definition-image\"]"
+    And  I click on "[data-cy=\"component-definition-money\"]"
+    Then I expect "[id^=\"image_\"]" exists
+    And  I expect "[id^=\"money_\"]" exists
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-label-leto-modelizer.config.json\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "default.plugin-test.js"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "leto-modelizer.config.json"
+    And  I wait 1 second
```

Comment:
and -> when
put it in the next paragraph

Reply:
done